packages/syft/src/syft/util/util.py

```diff
@@ -1082,3 +1082,49 @@ def get_nb_secrets(defaults: dict | None = None) -> dict:
         print(f"Unable to load {filename}")
 
     return defaults
+
+
+def repr_truncation(obj: Any, max_elements: int = 10) -> str:
```

Comment:
Looks great Khoa! So that this is a little standard, maybe we can use pprint.pformat with compact=True to get the truncated collection reprs?

Reply:
Thanks for the review @BrendanSchell, I tried with `pprint.pformat` but I think just using the function is not enough 
![image](https://github.com/user-attachments/assets/6f020992-19e2-49fe-b006-271034fe20d7)

Reply:
ya nvm, I think reprlib.repr is probably the one I was thinking of https://docs.python.org/3/library/reprlib.html

Reply:
Thanks @BrendanSchell
Indeed `reprlib.repr` provides a more general solution to this problem. I fixed the code accordingly